dotCMS/src/main/java/com/dotmarketing/filters/VanityURLFilter.java

```diff
@@ -88,7 +88,7 @@ public void doFilter(final ServletRequest req, final ServletResponse res,
 
                 //If the handler already resolved the requested URI we stop the processing here
                 if (vanityUrlResult.isResolved()) {
-                    DbConnectionFactory.closeSilently();
+                    DbConnectionFactory.closeSilently(); // todo: not sure if this is necessary
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)